Alignment/RPData/LHC/2015_10_18_fill4511/version2/sr+el/45.xml

```diff
@@ -0,0 +1,115 @@
+<!--
```

Comment:
I think Ianna refers to the name of the directory, `.../sr+el/...` .

That's not going to be an issue for web pages (browsers will replace the + with %2B), but it may be annoying for people cloning the CMSSW repository while using non-test-impaired filesystems...